src/idl_gen_csharp.cpp

```diff
@@ -947,7 +947,8 @@ class CSharpGenerator : public BaseGenerator {
       }
       // JVM specifications restrict default constructor params to be < 255.
       // Longs and doubles take up 2 units, so we set the limit to be < 127.
-      if (has_no_struct_fields && num_fields && num_fields < 127) {
+      if ((has_no_struct_fields || opts.generate_object_based_api) &&
```

Comment:
why is this object API only? I think the reason why we didn't do this before was because this code was shared with Java which doesn't have value structs.

Reply:
Hi,
thank you for your response. I would be happy if we could always generate the Create method but I thought we must Create the struct inline (e.g. CreateVec3() has to be called after StartMonster) and therefore need a way to call the Create method with the values needed to create the struct.
The Vec3T class I use for this is only generated for the object based api. Do you mean I should create a class like Vec3T always or a similar class?
If you can give me any hints I would be happy to update my pull request.

Thanks

btw. I added my email address to the CLA

Reply:
Ahh.. I guess we don't actually use C# structs to represent FlatBuffer structs, so you are probably right.

As for the CLA, you have to respond to this PR with `@googlebot I signed it!` (see above)